examples/xgboost/prepare_train_config.py

```diff
@@ -0,0 +1,90 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import json
+
+import numpy as np
+
+parser = argparse.ArgumentParser(description="generate train configs for HIGGS dataset")
+parser.add_argument("--site_num", type=int, default=5, help="Total number of sites")
+parser.add_argument("--site_name", type=str, default="site-", help="Site name prefix")
+parser.add_argument("--size_total", type=int, default=11000000, help="Total number of instances, default 11 million")
+parser.add_argument("--size_valid", type=int, default=1000000, help="Validation size, default 1 million")
+parser.add_argument("--split_method", type=str, default="uniform", help="How to split the dataset")
+parser.add_argument("--out_path", type=str, default="train_configs/config_train_uniform.json", help="Path to json file")
```

Comment:
can you explain these options in the README?

Reply:
added